models/repo_list.go

```diff
@@ -116,7 +117,7 @@ type SearchRepoOptions struct {
 	OwnerID         int64
 	PriorityOwnerID int64
 	TeamID          int64
-	OrderBy         db.SearchOrderBy
+	OrderBy         func(sess *xorm.Session) *xorm.Session
 	Private         bool // Include private repositories in results
 	StarredByID     int64
 	WatchedByID     int64
```

Comment:
What I just noticed: Why do we have two times the same constants?
Once in `db/search`, and once here in L172+?

(The comment is here because GitHub doesn't let me comment down below)